internal/cli/auth/login.go

```diff
@@ -48,12 +48,12 @@ type LoginConfig interface {
 }
 
 const (
-	AlreadyAuthenticatedError      = "you are already authenticated with an API key (Public key: %s)"
+	alreadyAuthenticatedError      = "you are already authenticated with an API key (Public key: %s)"
```

Comment:
reducing visibility, I have a follow up task to clean up these const seems they are only used in 1 place but it requires some changes to tests